app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class AddressSearchController < ApplicationController
+      include ArcgisApi
+
+      def geocoder
+        @geocoder ||= Geocoder.new
+      end
+
+      def suggest(address)
+        geocoder.suggest(address)
+      end
+
+      def find_address_candidates(magic_key)
+        geocoder.find_address_candidates(magic_key)
+      end
+
+      def index
```

Comment:
for organizaiton, typically we put the controller methods (`index` `show` `update` etc) at the top of the file and mark the rest of things as private